electron/src/runtime/lifecycle.ts

```diff
@@ -71,6 +71,10 @@ export const getWebViewId = (contents: WebContents): string | undefined => {
 export const quit = (): void => {
   logger.info('Initiating app quit ...');
   settings.persistToFile();
+  logger.info('Clear cache ...');
+  session.defaultSession.clearCache().catch(error => {
+    logger.info(error);
```

Comment:
Let's flag it as an error or at least a warning.

Reply:
done..